pytorch/pytorch_checkpoint.py

```diff
@@ -159,9 +158,15 @@ def objective(trial):
                 "optimizer_state_dict": optimizer.state_dict(),
                 "accuracy": accuracy,
             },
-            tmp_checkpoint_path,
+            "./tmp_model.pt",
```

Comment:
Could you change the path of checkpoint for each trial? If we run this script with multi-process, the saved models can be broken by other processes.